packages/fbtee/src/GenderConst.tsx

```diff
@@ -1,15 +1,8 @@
 enum GenderConst {
   NOT_A_PERSON = 0,
-  FEMALE_SINGULAR = 1,
-  MALE_SINGULAR = 2,
-  FEMALE_SINGULAR_GUESS = 3,
-  MALE_SINGULAR_GUESS = 4,
-  MIXED_UNKNOWN = 5,
-  NEUTER_SINGULAR = 6,
+  FEMALE = 1,
+  MALE = 2,
```

Comment:
I'd suggest to keep `MALE/FEMALE_SINGULAR` entries rather than renaming them.
The advantage of keeping that `_SINGULAR` suffix is that it's clear that these entries are for non-plural scenarios; I think that's important for fbt users who may not be familiar with the impact of choosing singular/plural cases for this enum.

Note also that the existing implementation redirects most plural cases like follow:

https://github.com/nkzw-tech/fbtee/blob/17f6c4d02f01faa6281130a7329a116abd319cc6/packages/fbtee/src/fbt.tsx#L46-L78

Reply:
I agree. Keeping `singular` in the name is good.